src/ExtUvLoop.php

```diff
@@ -0,0 +1,304 @@
+<?php
+
+namespace React\EventLoop;
+
+use React\EventLoop\Tick\FutureTickQueue;
+use React\EventLoop\Timer\Timer;
+use SplObjectStorage;
+
+/**
+ * @see https://github.com/bwoebi/php-uv
+ */
```

Comment:
This package is (now) available via PECL (https://pecl.php.net/package/uv), so this PR should probably be updated to use reference official `ext-uv` instead of installing from source :+1: Docblock and README should be in line with `ext-ev`.

Reply:
Completely missed that release on PECL, updated the docblock and README :+1: